packages/webapi/README.md

```diff
@@ -38,4 +38,17 @@ let taskStore = new Store<Workorder>();
 let taskService = new StoreApiService<Task>(taskStore);
 let taskRouter: express.Router = webapi(taskService, config);
 router.add("workflows", taskRouter);
-```
\ No newline at end of file
+```
+
+
+User API Endpoints
```

Comment:
Little bit misunderstanding. Webapi module is "generic module that allows users to build api in seconds"